api/ipam/ipprovider/helper.go

```diff
@@ -88,6 +88,13 @@ func (m *IpPrefix) Validate() error {
 	return nil
 }
 
+func (m *IpFamily) Validate() error {
+	if m.GetFamily() != IpFamily_IPV4 && m.GetFamily() != IpFamily_IPV6 {
```

Comment:
I would change this statement :) it seems convoluted. How about checking if the family is either IPV4 or IPV6 and return an error if not.

Reply:
updated